Automattic/studio
# Studio: Add events for import/export

Related to https://github.com/Automattic/studio/pull/365
Fixes:https://github.com/Automattic/dotcom-forge/issues/8364
Related to: https://github.com/Automattic/studio/pull/376

## Proposed Changes
This PR adds events to the importer/exporter to notify UI.


## Testing Instructions
1. Apply the following patch:
```patch
diff --git forkSrcPrefix/src/components/content-tab-import-export.tsx forkDstPrefix/src/components/content-tab-import-export.tsx
index 8bbdfff7d18e09def9de8416f8939ba32cca6645..ca844b82f8592d6d777f40f2092948888c3635bf 100644
--- forkSrcPrefix/src/components/content-tab-import-export.tsx
+++ forkDstPrefix/src/components/content-tab-import-export.tsx
@@ -270,9 +270,11 @@ const ImportSite = ( props: { selectedSite: SiteDetails } ) => {
 export function ContentTabImportExport( { selectedSite }: ContentTabImportExportProps ) {
 	useIpcListener( 'on-import', ( _evt, data: unknown ) => {
 		// This listener will be used to track progress of import when the UI is finished.
+		console.log( 'on-import-events', data );
 	} );
 	useIpcListener( 'on-export', ( _evt, data: unknown ) => {
 		// This listener will be used to track progress of export when the UI is finished.
+		console.log( 'on-export-events', data );
 	} );
 
 	return (

```
2. Start an export
3. Ensure that in console you see logs like below:
![2024-07-19T13:02:35,386393318+03:00](https://github.com/user-attachments/assets/30cb898f-d4ec-4f66-ba4c-f1d55657b443)
4. Start an import
5. Ensure that you see events in console like below:
![2024-07-19T13:03:01,028805078+03:00](https://github.com/user-attachments/assets/3bc0eea6-0a9a-4d94-9ccf-911b03dcd4ed)





## Pre-merge Checklist
